.github/sync-files.yaml

```
@@ -10,3 +10,8 @@
     - source: .prettierignore
     - source: .prettierrc.yaml
     - source: .yamllint.yaml
+
+- repository: autowarefoundation/autoware-github-actions
+  files:
+    - source: .github/workflows/github-release.yaml
```

Could you add this as well?
https://github.com/autowarefoundation/autoware-github-actions/pull/53

```suggestion
    - source: .github/workflows/actions-tagger.yaml
    - source: .github/workflows/github-release.yaml
```

---

Or is this not necessary for pre-commit hooks?